script.js

```
@@ -0,0 +1,258 @@
+// array to hold the current expression (numbers and operators), by default it has a number 0
+let mathExpressions = []; 
+// variable to hold the current number being typed
+let currentInput = ''; 
+// placeholder for future operator usage (not used in the current implementation)
+let operator = null;
+
+
+// function to check if a value is an operator
+/*
+input: a number or an operator
+process: 
+    - check if the parameter is an operator, or a number by the includes method
+    - if it's an operator return true, else return false
+output: a boolean value
+*/
+function isOperator(value) {
+    // returns true if the value is one of the four operators
+    return ['+', '-', '*', '/'].includes(value); 
+}
+
+
+// selects close error popup button
+const closePopupBtn = document.getElementById('close-popup-btn');
+// selects the error popup
+const errorPopup = document.getElementById('errorPopup');
+
+/*
+input: a method
+process: set up the error popup style as none to be hidden because the user want to stop seeing the popup
+ouput: none
+*/
+closePopupBtn.addEventListener('click', () => {
+    // set up the style the element which has its id as "errorPopup" as "none"
```

CS5.2: chỗ này em có thể comment đơn giản và ngắn gọn hơn ví dụ như `Hide the error popup`

---

unresolved comment

---

unresolved comment